hack/create-release-pr.sh

```diff
@@ -40,10 +40,22 @@ fi
 
 echo $NEW_VERSION > VERSION
 IMAGE_TAG="v${NEW_VERSION}"
+# Update manifests
 make manifests
 
+# Create PR for the release
 git checkout -b "feat/new-version-${NEW_VERSION}"
-git commit -m "Release ${NEW_VERSION}" VERSION manifests/
+
+# Create tag for registry-scanner
+git tag -a ${IMAGE_TAG} -m "Registry-Scanner Release ${IMAGE_TAG}"
```

Comment:
I remember reading somewhere that tagging a sub-module needs to have the sub-module name as prefix, something like `registry-scanner/v0.16.0` as the tag name. Otherwise, one can't tell if this tag is for the main module, or any sub-module.